backend/src/akvo/lumen/admin/add_tenant.clj

```diff
@@ -26,11 +26,11 @@
   (:require [akvo.lumen.admin.util :as util]
             [akvo.lumen.component.keycloak :as keycloak]
             [akvo.lumen.config :refer [error-msg]]
+            [akvo.lumen.http :as http]
```

Comment:
maybe http-client is a better name?

Reply:
the alias?

Reply:
or the namespace?

Reply:
http namespace is wrapping http-connection-manager things and http-client things ... so i think keeping generic is better

Reply:
I didn't realize about `server` world ... I'll move the ns to http.client

Reply:
solved by https://github.com/akvo/akvo-lumen/pull/2227/commits/781c970b52e8282a5abff5c79ce677708215cab5